services/QuillLMS/client/app/bundles/Shared/libs/pageTitle.ts

```diff
@@ -0,0 +1,7 @@
+export const getPageTitle = (primaryLabel: string, secondaryLabel?: string) => {
```

Comment:
Love making this into a reusable method. I wonder if we need the second argument, or just to pass the title with the `|` if there's a secondary? I see that we're already doing that with tertiary ones in some places. NBD either way to me.

Reply:
it was a bit verbose in some cases, and I also liked having the hierarchal separation